<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDockWidget</class>
 <widget class="QDockWidget" name="MainDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>199</width>
    <height>200</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>Render Settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="raytraceButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>191</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>175</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ray Trace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sceneFileBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scene File:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="subDivSpinner">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sample Sub Div Per Pixel:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="resolutionXSpinner">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1280</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="resolutionYSpinner">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>720</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lens Width:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lensWidthSpinner">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="focalLengthSpinner">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Focal Length:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sceneBrowseButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>69</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageSaveBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>resolutionXSpinner</tabstop>
  <tabstop>resolutionYSpinner</tabstop>
  <tabstop>lensWidthSpinner</tabstop>
  <tabstop>focalLengthSpinner</tabstop>
  <tabstop>subDivSpinner</tabstop>
  <tabstop>sceneFileBox</tabstop>
  <tabstop>sceneBrowseButton</tabstop>
  <tabstop>raytraceButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
